site/docs/transform/filter.md

```diff
@@ -30,13 +30,12 @@ Vega-Lite filter transforms must have the `filter` property.
 For a [Vega Expression](https://vega.github.io/vega/docs/expressions/) string, each datum object can be referred using bound variable `datum`. For example, setting `filter` to `"datum.b2 > 60"` would make the output data includes only items that have values in the field `b2` over 60.
 
 
-## Filter Object
+## Field Predicate Object
```

Comment:
This is a bit inconsistent. Below, we call the filter "Selection Filter". I'd prefer to keep the name (it is also less confusing) and say that the "Filter Object" has to contain a "Filter Predicate". Alternatively, we could name it "Field Filter Object". The predicate is the expression that is used to filter.

Reply:
I will change selection filter to selection predicate.  We should not call them filters because they are also use for condition syntax in a follow-up PR.  Field predicate is also more applicable in more places so we need to distinguish them.  The old text is confusing anyway.